src/olympia/reviewers/tests/test_views.py

```diff
@@ -3016,6 +3016,33 @@ def test_item_history(self, channel=amo.RELEASE_CHANNEL_LISTED):
             assert ((reviewer_name == self.reviewer.name) or
                     (reviewer_name == self.other_reviewer.name))
 
+    def test_item_history_pagination(self):
```

Comment:
surprised the existing tests didn't pick this up, tbh

Reply:
Yep. I looked, and this didn't appear to be covered by any tests - all pagination related tests are testing the queues